genesis/src/main.rs

```diff
@@ -85,16 +83,30 @@ fn main() -> Result<(), Box<dyn error::Error>> {
     let bootstrap_vote_keypair = read_keypair(bootstrap_vote_keypair_file)?;
     let mint_keypair = read_keypair(mint_keypair_file)?;
 
-    let (mut genesis_block, _mint_keypair) = GenesisBlock::new_with_leader(
-        lamports,
+    let genesis_block = GenesisBlock::new(
```

Comment:
I would have implemented this in the builder pattern, but I ran out of steam... TODO ;)